### DISCUSSION TITLE
Warnings on deprecated linters like `execinquery`

### DISCUSSION THREAD

**Discussion by gkech** [golangci/golangci-lint#4858] [Q&A] [answered]
Hello,

Noticed that a warning for a deprecated linter started to appear every time I'm running`golangci-lint run`:

`level=warning msg="The linter 'execinquery' is deprecated (since v1.58.0) due to: The repository of the linter has been archived by the owner. "
`

I'm wondering if this linter (https://github.com/1uf3/execinquery) is planned to be removed from the project given that it is deprecated and if yes, what's the timeline with that. Is there a policy on how deprecated linters are managed within the golanci-lint project? 

Thanks a lot

(edit) PS: I'm currently using the version `v1.59.1`

**Comment by ldez** [comment, marked as answer]
Hello,

the deprecation cycle of a linter is explained here: https://golangci-lint.run/product/roadmap/#linter-deprecation-cycle

**Reply by gkech** [reply]
thanks a lot @ldez

**Reply by thusharopen** [reply]
Can I know the alternative of this linter?
